lib/log.js

```diff
@@ -0,0 +1,8 @@
+'use srtict';
```

Comment:
Created a log wrapper that will print `tke-cron-master-builder-$JOB_NAME` as a prefix instead of a file name